types/jsrsasign/modules/KJUR/asn1/csr/CSRUtil.d.ts

```diff
@@ -103,5 +111,41 @@ declare namespace jsrsasign.KJUR.asn1.csr {
          * console.log(o.subject.name) → "/C=US/O=Test"
          */
         function getInfo(sPEM: string): PEMInfo;
+
+        /**
+         * get field values from CSR/PKCS#10 PEM string
+         * @param sPEM PEM string of CSR/PKCS#10
+         * @returns JSON object with parsed parameters such as name or public key
+         * @description
+         * This method parses PEM CSR/PKCS#1 string and retrieves
+         * fields such as subject name and public key.
+         * Following parameters are available in the
+         * resulted JSON object.
+         *
+         * - {X500Name}subject - subject name parameters
```

Comment:
Might be worth moving this comment to the ParamResponse interface? But also appreciate may want to keep this together given it matches the docs.